lib/Providers/Qr/BaconQrCodeProvider.php

```diff
@@ -23,10 +22,10 @@ class BaconQrCodeProvider implements IQRCodeProvider
      * Ensure we using the latest Bacon QR Code and specify default options
      */
     public function __construct(
-        private int $borderWidth = 4,
+        private readonly int $borderWidth = 4,
```

Comment:
I don't think there is much point in making private properties readonly as they can only be used within this class so it feels like a waste of effort.

Also why aren't the other properties defined as readonly? What makes this one deserving of readonly and not the others?

Reply:
`borderWidth` is the only variable here that doesnt get modified anywhere in the code. The rest of the properties are overwritten in the constructor by various methods.